lesson12/index.html

```diff
@@ -1 +1,18 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+   <title>lesson12</title>
+   <link rel="stylesheet" href="style.css">
+</head>
+<body id="body">
+   <ul id="lists">
+      <button id="create-element-btn">Click Me</button>
+   </ul>
```

Comment:
【IMO】
I think the button is outside of ul is better. Because it is not part of ul.

Reply:
Thank you for your review.
I changed the place of that button.
5db123a